internal/db/service.go

```diff
@@ -7,10 +7,24 @@
 
 package db
 
-import "database/sql"
+import (
+	"database/sql"
+	"github.com/patrickwilmes/goauth/internal/common"
+)
+
+const (
+	dbDriverName = "database.driverName"
+	dbUrl        = "database.url"
+)
 
 type DatabaseBackend interface {
 	Exec(sql string) error
 	Database() *sql.DB
 	Close() error
 }
+
+func CreateDatabaseBackend() (DatabaseBackend, error) {
+	driverName := common.GetStringValue(dbDriverName)
```

Comment:
What happens if we have an undefined / wrong driver name? Won't everything just crash and burn, I wonder? 

```suggestion
	driverName := common.GetStringValue(dbDriverName)
	if driverName != "sqlite3" {
	    log.Error("Unknown driver!")
	}
```

Maybe something like that. I guess we can directly equal check the driver names, can't we? Let's also log if the driver is not supported.

Reply:
I added the check for driver and an error will be thrown for unsupported drivers. I use standard log of golang and simply print the message to standard out. We have to talk about a logging strategy for the service because log did not support different log levels.